Automattic/jetpack
# Shortcodes: remove Lytro shortcode and add it to unavailable list.

#### Changes proposed in this Pull Request:

- Lytro was shut down in March 2019.
- Also took the opportunity to clear all phpcs warnings for the unavailable.php file.

D25302-code will bring those 2 files in sync with WordPress.com.

#### Testing instructions:

* Add the following to a post: `[lytro photo='202' show_arrow='true' show_border='true' show_first_time_user='true' allow_full_view='true']`
* View that post on the frontend: you should see an error message showing that the service has now shut down.

#### Proposed changelog entry for your changes:

* Shortcodes: remove Lytro service.
